### ISSUE TITLE
Argument #2 ($value) must be of type array, Livewire\TemporaryUploadedFile given

### ISSUE THREAD

**Issue by ralphjsmit** [laravel-filament/filament#1049]
In response to #1037.

I just noticed that the `2.7.4` version gives another error:

```
TypeError : Filament\Forms\Components\BaseFileUpload::Filament\Forms\Components\{closure}(): Argument #2 ($value) must be of type array, Livewire\TemporaryUploadedFile given
```

The first error only happened when I set the image property to `null`, the second error happens when I set the image property to an `UploadedFile`.

_Originally posted by @ralphjsmit in https://github.com/laravel-filament/filament/issues/1037#issuecomment-1002552014_

**Comment by robertorinaldi-dev** [comment]
Maybe I could make a fix for this tomorrow, but I'm not sure it has to be fixed. BaseFileUpload needs an array, it should work if you set [$file]

**Comment by danharrin** [comment]
Should be fixed by #1063, let me know if not.

**Comment by ralphjsmit** [comment]
Yes, is fixed, great!
